crates/gitbutler-core/src/ops/oplog.rs

```diff
@@ -349,7 +350,7 @@ impl Oplog for Project {
                     lines_added,
                     lines_removed,
                     files_changed,
-                    created_at: commit.time().seconds(),
+                    created_at: Duration::from_secs(commit.time().seconds().try_into().unwrap()),
```

Comment:
What started as a strong recommendation now becomes a request as this is incorrect. `time().seconds()` is `i64` which is now cast into `u64`. It's a feature of `git2` (and `gitoxide`) to be able to represent commit-dates older than 1970 to allow the creation of 'historic' repositories (used for legal-texts apparently), and this `try_into()` will panic there.